app/src/Game.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Board from './Board';
+
+export const TurnContext = React.createContext();
+export const SetTurnContext = React.createContext();
+
+const Game = () => {
+	const [turn, setTurn] = React.useState('X');
+
+	return (
+		<div>
+			<h1>Player {turn}, its your turn</h1>
+			<TurnContext.Provider value={turn}>
```

Comment:
It's cool that you used a context here, because it demonstrates knowledge of different React hooks. Nice work! One thing to think about is - when you're making a context, it might make sense to group related pieces of data together. For this one, I'd probably include a bunch of the game data on the context so that it is all in one place. This could include:

1. The board
2. The current player
3. The history of the board
4. Whether or not we have a winner
5. A function for the current player to take their turn

You have all of that stuff in your app, but it's just spread out. Keeping it all in the context could help keep the components a little bit smaller.